src/shared/prosemirror-plugins/image-upload.ts

```diff
@@ -411,7 +438,12 @@ export class ImageUploader extends PluginInterfaceView<
                 return;
             case ValidationResult.InvalidFileType:
                 this.showValidationError(
-                    _t("image_upload.upload_error_unsupported_format")
+                    _t("image_upload.upload_error_unsupported_format", {
```

Comment:
I definitely missed this validation bit for the updated string, thank you

Reply:
No problem!